admin/lint/checks.go

```diff
@@ -31,7 +31,7 @@ import (
 
 // checkLicense checks if the given file contains the necessary license information and returns an
 // error if this is not true or if the check cannot be performed.
-func checkLicense(file string) error {
+func checkLicense(workspaceDir string, file string) error {
```

Comment:
unrelated changes?

Reply:
No, these were necessary. The workspace directory changes were already in a separate commit with a justification for them (to fix Gazelle runs) and I have now also split out the test changes to explain why they are necessary (to pass with Go 1.11).